src/operation/get-parameters-as-json-schema.ts

```diff
@@ -330,15 +300,17 @@ export default function getParametersAsJsonSchema(
     });
   }
 
-  const hasRequestBody = !!operation.requestBody;
-  const hasParameters = !!(operation.parameters && operation.parameters.length !== 0);
-  if (!hasParameters && !hasRequestBody && getCommonParams().length === 0) return null;
+  // If this operation neither has any parameters or a request body then we should return null because there won't be
+  // any JSON Schema.
+  if (!operation.hasParameters() && !operation.hasRequestBody()) {
+    return null;
+  }
 
-  const components = getComponents();
+  const components = transformComponents();
 
   const typeKeys = Object.keys(types);
-  return [getRequestBody()]
-    .concat(...getParameters())
+  return [transformRequestBody()]
+    .concat(...transformParameters())
```

Comment:
Renamed all these `get*` methods to be `transform*` so it's clearer what they're doing. When I was grepping around for `getParameters()` usages while working on that method it was confusing to see this in the mix.